ibc_data/ibc_conditions.tsv

```diff
@@ -0,0 +1,380 @@
+SelectiveStopSignal	go_noncritical	Answer to the visual stimulus (noncritical side)
+SelectiveStopSignal	stop	Hold motor response
+SelectiveStopSignal	ignore	Answer regardless of the stop signal
+Stroop	congruent	Color and word are the same
+Stroop	incongruent	Color and word are different
+ColumbiaCards	num_loss_cards	Probability of losing in gambling
+ColumbiaCards	loss	Expected loss in gambling
+ColumbiaCards	gain	Expected gain in gambling
+DotPatterns	cue	Look at the stimulus to provide a description of the cue
+DotPatterns	correct_cue_correct_probe	Target pair, captures expectancy after correct cue
+DotPatterns	correct_cue_incorrect_probe	Nontarget pair that also captures expectancy after correct cue
+DotPatterns	incorrect_cue_correct_probe	Incorrect pair. The probe is correct but the cue is not
+DotPatterns	incorrect_cue_incorrect_probe	Incorrect pair, both are incorrect
+WardAndAllport	planning_ambiguous_intermediate	Partially ambiguous goal state with intermediate movement
+WardAndAllport	planning_unambiguous_direct	Unambiguous goal state without intermediate movement
+WardAndAllport	planning_ambiguous_direct	Partially ambiguous goal state without intermediate movement
+WardAndAllport	planning_unambiguous_intermediate	Unambiguous goal state with intermediate movement
+BiologicalMotion1	global_upright	"Structural information is preserved, but individual local trajectories are mirror-symmetric. ""Global-only"" in the original paper."
+BiologicalMotion1	global_inverted	Global walker, inverted upside-down
+BiologicalMotion1	natural_upright	"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion1	natural_inverted	Local natural walker, inverted along the horizontal axis
+BiologicalMotion2	modified_upright	"Neither structural or local information is carried out by this type of walker, it uses both types of modifications used for the previous two categories. ""Local modified"" in the original study."
+BiologicalMotion2	modified_inverted	Local modified walker, inverted along the horizontal axis
+BiologicalMotion2	natural_upright	"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion2	natural_inverted	Local natural walker, inverted along the horizontal axis
+MathLanguage	arithmetic_fact_auditory	Listen to arithmetic fact
+MathLanguage	arithmetic_fact_visual	Read arithmetic fact
+MathLanguage	arithmetic_principle_auditory	Listen to arithmetic principle
+MathLanguage	arithmetic_principle_visual	Read arithmetic principle
+MathLanguage	colorlessg_auditory	Jabberwocky sentence presented as auditory stimulus
+MathLanguage	colorlessg_visual	Jabberwocky sentence presented as visual stimulus
+MathLanguage	context_auditory	Fixation period
```

Comment:
Are you sure about lines 233-234 ? This feels wrong, but I'm not sure.

Reply:
I ran it, so at the beginning of each trial the subject is presented with either a red cross or hear a beep, indicating that the following stimuli will be auditory or visual, respectively. So it's just as context, with no expected response. However I agree that it sounds wrong, what about this:
`context_auditory:` Beeping sound indicating that the following stimuli will be audio
`context_visual:` Red cross indicating that the following stimuli will be visual

Reply:
Sounds good ! Please add that and then we can merge.